gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Channelizer.java

```diff
@@ -53,9 +57,16 @@
 
     /**
      * Initializes the {@code Channelizer}. Called just after construction.
+     * @param connection
+     *
+     * This method is deprecated since 3.4.2/3.3.7 and has been replaces by
```

Comment:
nit: could you please use this form for deprecation in the javadoc: https://github.com/apache/tinkerpop/blob/master/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/Graph.java#L306

Reply:
Changed.